framework/12-inch/13th-gen-intel/README.md

```diff
@@ -0,0 +1,17 @@
+# [Framework Laptop 12](https://frame.work/laptop12)
+
+## Updating Firmware
+
+First put enable `fwupd`
+
+```nix
+services.fwupd.enable = true;
+```
+
+Then run
+
+```sh
+ $ fwupdmgr update
+```
+
+- [Latest Update](https://fwupd.org/lvfs/devices/work.frame.Laptop12.RPL.BIOS.firmware)
```

Comment:
This link still returns a 404. Is it not published yet?

Reply:
Correct